src/main.rs

```diff
@@ -0,0 +1,61 @@
+use rustls::crypto::hmac::Hmac;
+use rustls_symcrypt::ecdh::*;
+/// TEMP FILE FOR TESTING DO NOT REVIEW
```

Comment:
If you wanted to make this file a little less temporary, you could make it an example: https://doc.rust-lang.org/cargo/reference/cargo-targets.html#examples

That could be helpful for showing people how to use the APIs.

Reply:
This file is just a local file used for testing will remove from the PR